camera_calibration_parsers/src/parse_yml.cpp.unc-backup~

```diff
@@ -0,0 +1,302 @@
+/*********************************************************************
```

Comment:
It looks like you've committed a backup file created by your editor. Please remove it.